### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by NathanBnm**
This PR comes in replacement of #6. I couldn't manage to set up correctly the commit signature but it should be right now.

### CODE REVIEW

**Review by ryonakano — CHANGES_REQUESTED** [review]
You seem to remove necessary files like `POTFILES`, `meson.build` and my Japanese translation :rofl:

**Comment by NathanBnm** [comment]
Whoops I didn't mind to do that.
I'm gonna fix this asap. Thanks.

**Comment by NathanBnm** [comment]
Don't know what happened 👀 Will fix soon

**Comment by ryonakano** [comment]
Almost all problems seem to be fixed.

Nitpicking but is the following change intentional?

Screenshot from 2020-01-31 21-27-29

It seems you're renaming the translation template file to the French translation file.

**Comment by NathanBnm** [comment]
Wtf this is messed up :sweat_smile: I'm sorry. Now hopefully everything should be fine!

File: po/extra/ja.po [hunk 1]
```diff
@@ -94,3 +94,4 @@ msgstr "com.github.eudaldgr.elements"
 #: data/com.github.eudaldgr.elements.desktop.in:12
 msgid "Science;Chemistry;Elements;"
 msgstr "Science;Chemistry;Elements;科学;サイエンス;化学;元素;"
+https://www.qwant.com/?client=ext-firefox-hp
```
Comment (ryonakano) [inline po/extra/ja.po]:
```suggestion
```

File: po/extra/POTFILES [hunk 2]
```diff
@@ -1,2 +1,3 @@
+
```
Comment (ryonakano) [inline po/extra/POTFILES]:
```suggestion
```

**Review by ryonakano — CHANGES_REQUESTED** [review]
Two more nitpicks :sweat:

**Pull request merged**